3D-microservices-observability/go-chi/services/delivery/delivery.go

```diff
@@ -0,0 +1,108 @@
+// 2019. Mahesh Voleti (mvoleti@vmware.com)
+
+package delivery
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+
+	. "wavefront.com/hackathon/beachshirts"
+)
+
+type DeliveryServer struct {
+	HostURL       string
+	Router        *chi.Mux
+	DeliveryQueue chan PackedShirts
+}
+
+func NewServer() *DeliveryServer {
```

Comment:
No need to export `DeliveryServer`. Can rename to `deliveryServer` and just return a handle to a `Server` interface.